Kernel/Process.cpp

```diff
@@ -3053,18 +3053,33 @@ int Process::sys$select(const Syscall::SC_select_params* params)
     return marked_fd_count;
 }
 
-int Process::sys$poll(pollfd* fds, int nfds, int timeout)
+int Process::sys$poll(const Syscall::SC_poll_params* params)
 {
     REQUIRE_PROMISE(stdio);
-    if (!validate_read_typed(fds))
```

Comment:
I think this should've passed nfds as 2nd parameter.

Reply:
Yes, you are right!